system/jlib/jlzw.cpp

```diff
@@ -2268,6 +2268,8 @@ ICompressedFileIO *createCompressedFileWriter(IFileIO *fileio,size32_t recordsiz
         trailer.crc = ~0U;
         trailer.compressedType = fast?FASTCOMPRESSEDFILEFLAG:COMPRESSEDFILEFLAG;
         trailer.blockSize = COMPRESSEDFILEBLOCKSIZE;
+        if (fast)
+            trailer.blockSize *= 4;
```

Comment:
A comment would be useful here as to why fast needs/is better with * 4 block size, vs non-fast not.